pkgs/tools/networking/wakeonlan/default.nix

```diff
@@ -13,6 +13,7 @@ perlPackages.buildPerlPackage rec {
 
   outputs = [ "out" ];
 
+  buildInputs = lib.optional stdenv.isDarwin shortenPerlShebang;
```

Comment:
Shouldn't this go to nativeBuildInputs?

Reply:
That's not how the [manual](https://nixos.org/manual/nixpkgs/unstable/#ssec-perl-packaging) prescribes it. That said, I do deviate from the manual anyway because it didn't work for me when I put the invocation in `postInstall`.

Reply:
that seems to be wrong but perl is not that cross friendly anyway

Reply:
Looks like it's time to fix the manual.

Reply:
I'll raise another PR to update the manual and start a discussion there.

For now, I've moved it to `nativeBuildInputs` and tested on both darwin & NixOS, it works on both.